client/securedrop_client/export.py

```diff
@@ -240,19 +240,28 @@ def _on_print_preflight_complete(self) -> None:
         """
         self._cleanup_tmpdir()
         if self.process:
-            output = self.process.readAllStandardError().data().decode("utf-8").strip()
+            output_untrusted = self.process.readAllStandardError().data().decode("utf-8").strip()
             try:
-                status = ExportStatus(output)
-                if status == ExportStatus.PRINT_PREFLIGHT_SUCCESS:
-                    self.print_preflight_check_succeeded.emit(status)
-                    logger.debug("Print preflight success")
+                if output_untrusted:
+                    logger.debug(f"Result is {output_untrusted}")
+
+                    # The final line of stderr is the status.
```

Comment:
It would be good to document what could come before this and why we're discarding it.

Reply:
Yeah that's fair. It's like https://github.com/freedomofpress/securedrop-client/blob/937483afb09df86185a308a35ae099d6fc45c3e5/client/securedrop_client/export.py#L211-L212 (comment farther down in the file) so I was a bit sparse on the documentation. Basically, we know that our status string is the last thing written to stderr, but because we flush stderr there could be other stuff written beforehand (by the OS/another process, not by us) that we don't want to pass on to the calling VM. The errors are written to the appropriate log (or to the journal) in sd-devices, but here, we're only interested in the status value.